tmt/steps/prepare/ansible.py

```diff
@@ -127,7 +127,8 @@ class PrepareAnsible(tmt.steps.prepare.PreparePlugin[PrepareAnsibleData]):
         prepare --how ansible --playbook ansible/packages.yml
 
     Run multiple playbooks in one phase, with extra arguments for
-    ``ansible-playbook``:
+    ``ansible-playbook``. Each of them will be applied in the given order.
+    The path must be relative to the metadata tree root.
```

Comment:
The information about metadata root is already included in the `warning` on line 95, and the order is pretty much implied by `playbook` being a list.

I'd say `prepare/ansible` is fine, no information is missing, so it's just a question of removing those lines from `spec/plans/prepare.fmf`, nothing more to do in the plugin itself.

Reply:
That's right, sorry for that I didn't saw the warning, changed for ansible undone !